nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,147 @@
+     create a custom setup with a window manager, you can select "No desktop".
+
+     ::: {.tip}
+     If you don't have a favorite desktop and don't know which one to choose,
+     you can stick to either GNOME or Plasma. They have a quite different
+     design, so you should be able to choose which one you like better.
+     They are both popular choices and well tested on NixOS.
+     :::
+
+6.   You have the option to allow unfree software in the next screen.
+
+7.   The easiest option in the "Partitioning" screen is "Erase disk", which will
+     delete all data from the selected disk and install the system on it.
+     Also select "Swap (with Hibernation)" in the dropdown below it.
+     You have the option to encrypt the whole disk with LUKS.
+
+     ::: {.note}
+     At the top left you see if the Installer was booted with BIOS or UEFI. If
+     you know your system supports UEFI and it shows "BIOS", reboot with the
+     correct option.
+     :::
+
+     ::: {.warning}
+     Make sure you have selected the correct disk at the top and that no valuable
+     data is still on the disk!
+     :::
+
+8.   Check the choices you made in the "Summary" and click "Install".
+
+     ::: {.note}
+     The install takes about 15 minutes. The time varies based on the selected
+     desktop, internet speed and disk write speed.
+     :::
+
+9.  When the install is complete, reboot into your new system!
 
-NixOS can be installed on BIOS or UEFI systems. The procedure for a UEFI
-installation is by and large the same as a BIOS installation. The
-differences are mentioned in the steps that follow.
+## Manual Installation {#sec-installation-manual}
 
-The installation media can be burned to a CD, or now more commonly,
-"burned" to a USB drive (see [](#sec-booting-from-usb)).
+NixOS can be installed on BIOS or UEFI systems. The procedure for a UEFI
+installation is broadly the same as for a BIOS installation. The differences
+are mentioned in the following steps.
 
-The installation media contains a basic NixOS installation. When it's
-finished booting, it should have detected most of your hardware.
+The NixOS manual is available by running `nixos-help` in the command line
+or from the application menu in the desktop environment.
 
-The NixOS manual is available by running `nixos-help`.
+To have access to the command line on the graphical images, open
+Terminal (GNOME) or Konsole (Plasma) from the application menu.
 
 You are logged-in automatically as `nixos`. The `nixos` user account has
 an empty password so you can use `sudo` without a password:
+
 ```ShellSession
 $ sudo -i
 ```
 
-If you downloaded the graphical ISO image, you can run `systemctl
-start display-manager` to start the desktop environment. If you want
-to continue on the terminal, you can use `loadkeys` to switch to your
-preferred keyboard layout. (We even provide neo2 via `loadkeys de
-neo`!)
+You can use `loadkeys` to switch to your preferred keyboard layout.
```

Comment:
Is there a difference to setxkbmap?

Reply:
Yes.

`loadkeys` works on the kernel fbcon. `setxkbmap` works on X11.

Reply:
i have no idea. i have not changed the command, so maybe discuss that in a new issue

it does work, so i don't know if we should change it